pug/test_viewResult.pug

```diff
@@ -71,7 +86,39 @@ block content
       else
         dt span.alert.alert-danger Component information is not available!
     
-    pre #{''}
+      pre #{''}
+        
+      if(formrec.formName == "Wire Tensions")
+        .row
+          .col-9
+            div.border-warning.border.rounded.p-2
+              .row
+                .col <strong>Query a Wire Tension: </strong>
+                .col
+                  select.form-control(id = "wireLayer", type = "text")
+                    option(value = '' ) Wire Layer
+                    option(value = 'X') X
+                    option(value = 'U') U
+                    option(value = 'V') V
+                    option(value = 'G') G
+                .col
+                  select.form-control(id = "apaSide", type = "text")
+                    option(value = '' ) APA Side
+                    option(value = 'A') A
+                    option(value = 'B') B
+                .col
+                  input.form-control(id = "wireNumber", type = "text", placeholder = "Wire Number")
+                .col
+                  input.btn.btn-info#submitPromotion(type = "submit", onclick = "GetWireTension()")
+                .col-2
+                  input.form-control(id = "tensionKey", type = "text", placeholder = "Tension Key", readonly)
+                .col-2
+                  input.form-control(id = "wireTension", type = "text", placeholder = "Wire Tension (N)", readonly)
+          
+          .col-3
+      
+        pre #{''}
+        pre #{''}
```

Comment:
Why are preformatted elements used here?

Reply:
I've found that it's the best way to just get some spacing between elements and at the bottom of pages ... most of the other methods that I've tried don't consistently work for whatever reason.

Reply:
Fair enough.  Now, baring in mind that I've done the exact same thing in every other pug file that I've looked over in the past 2 months, can we leave this here for this commit and I will put in a separate commit where I scrub all of the pre statements in favour of the correct code?  Would like to have some time to try out what works and what doesn't, rather than rushing a change for this commit.